src/main/java/org/homework/step4/view/InputView.java

```diff
@@ -0,0 +1,116 @@
+package org.homework.step4.view;
+
+import org.homework.step4.dto.CreateTodoDTO;
+import org.homework.step4.enums.Menu;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class InputView {
+
+    private final Scanner scan = new Scanner(System.in);
+
+    private Integer toInteger(String value) {
+        try {
+            return Integer.valueOf(value);
+        } catch (NumberFormatException numberFormatException) {
+            throw new NumberFormatException("잘못된 입력값 입니다");
+        }
+    }
+
+    private int checkValueNegative(int value) {
+        if (value < 0) {
+            throw new InputMismatchException();
+        }
+        return value;
+    }
+
+    public int getUpdatedTodoId() {
+        while (true) {
+            System.out.print("수정할 ID: ");
+            try {
+                return Integer.parseInt(this.getLine());
+            } catch (NumberFormatException e) {
+                System.out.println("올바른 ID를 입력해 주세요");
+            }
+        }
+    }
```

Comment:
같은 코드가 너무 반복된다면 메소드로 묶어도 좋습니다.

```suggestion
    public int getUpdatedTodoId() {
        System.out.print("수정할 ID: ");
        
        // while 단위로 메소드로 묶어도 좋고
        while (true) {
           // parse 부분만 묶어도 좋습니다.
            try {
                return Integer.parseInt(this.getLine());
            } catch (NumberFormatException e) {
                System.out.println("올바른 ID를 입력해 주세요");
            }
        }
    }
```